cmd/vicadmin/server.go

```diff
@@ -266,7 +274,7 @@ func (s *server) loginPage(res http.ResponseWriter, req *http.Request) {
 		if err != nil || usersession == nil {
 			// something went wrong or we could not authenticate
 			log.Warnf("User %s from %s failed to authenticated at %s", user, req.RemoteAddr, time.Now())
-			http.Error(res, "Authentication failed due to incorrect credential(s)", 400)
+			http.Error(res, "Authentication failed due to incorrect credential(s)", http.StatusBadRequest)
```

Comment:
I changed this to `http.StatusBadRequest` solely to remove the hardcoded value. Would using `http.StatusUnauthorized` be more appropriate?